.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: build and push docker image
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.0.3
+      - uses: actions/cache@v1
+        with:
+          path: ~/operator-sdk/bin/
+          key: operator-sdk-${{ env.OPERATOR_SDK_VERSION }}
```

Comment:
Where does `OPERATOR_SDK_VERSION` actually come from? I see that it is set explicitly in `pre-commit.yml`, but what about here?

Reply:
copy&paste error

Reply:
fixed